src/pages/admins/team.tsx

```diff
@@ -138,29 +139,11 @@ const AdminCard: React.FC<Admin> = ({
 };
 
 const AdminTeam = () => {
-  return (
-    <div>
-      <Header />
-      <div className='flex justify-center items-center'>
-        <div className='w-[100%] md:w-fit p-4 m-4 justify-center rounded-md shadow-md bg-slate-200 dark:bg-slate-800'>
-          <div className='flex flex-col items-center justify-center space-y-5 mb-4 sm:space-y-4 md:max-w-xl lg:max-w-3xl xl:max-w-none'>
-            <h2 className='text-3xl font-bold dark:text-slate-200 sm:text-4xl text-center'>
-              Il team di Latina In Tech
-            </h2>
-            <p className='mx-auto max-w-2xl text-m text-center text-gray-500 dark:text-slate-400 sm:mt-2'>
-              Un gruppo di persone che condividono la passione per la tecnologia
-              e per il proprio territorio.
-            </p>
-          </div>
-          <div className='grid grid-cols-1 justify-items-center md:grid-cols-3 lg:grid-cols-4 gap-4'>
-            {admins.map(admin => (
-              <AdminCard key={admin.name} {...admin} />
-            ))}
-          </div>
-        </div>
-      </div>
-    </div>
-  );
+  useEffect(() => {
+    window.location.href = `/${i18n.defaultLocale}`;
```

Comment:
Actually it was me to suggest him to leave the old routes and re-direct from them to the new ones 😅 Do you think it is fine to break the URLs? ALso another thing I can think of, when you type an exact URL on a big site, you will not get a 404, but you will be just redirected to the default language.

Perhaps we can use a query parameter instead of the path itself? i.e. `/admins/team?language=en` instead of `/en/admins/team`

Reply:
ok, it makes sense. I was not considering the event pages that are been spread in several sites/posts
I'm wondering if we can have a clearer solution in order to avoid components that render nothing (it's a kind of anti pattern) and code duplication (I'm figuring a new contributor that meets a duplicated resources for each resource in pages directory)

What about to use a backward compatibility hook in the `App` component?
In that way we do one thing in one place and we can remove it when we'll be pretty sure users can't access old ulrs

something like this

```ts
const oldPaths = [
  '/admins/team',
  '/events/\\d+',
  '/feedback/new',
  '/newsletter',
  '/community'
].map(p => new RegExp(`^${p}`));
const useBackRouteCompatibility = () => {
  const router = useRouter();
  React.useEffect(() => {
    if (oldPaths.some(p => p.test(router.asPath))) {
      void router.replace(
        router.asPath,
        `/${i18n.defaultLocale}${router.asPath}`
      );
    }
  }, [router]);
};

[...]

export default function App({ Component, pageProps }: AppProps) {
  useBackRouteCompatibility();
  return (
      <Component {...pageProps} />
  );
}

```
---

btw, no strictly related this, with this implementation if a user navigates LiT sites and click on an event, he will be redirected always to the home page

https://github.com/latina-in-tech/latina-in-tech.github.io/assets/822471/1c288bc5-a3f7-40c2-b4bd-4cb3a3689664


---